### ISSUE TITLE
Accept water_heater entities in climate card

### ISSUE THREAD

**Issue by AlessandroTischer** [piitaya/lovelace-mushroom#788]
### Is your feature request related to a problem?

I would like to add a water_heater entity into my dashboard made with mashrooms cards, but I can add it only as an entity card

### Describe the solution you'd like

Accept water_heater domain for climate card

### Describe alternatives you've considered

Add a water_heater card equivalent to the climate one

**Comment by yestrdave** [comment]
I would agree I tried to use the climate card to control my pool and hot tub heater everything works except the temperature control value does not update the desired temperature of the pool.

type: custom:mushroom-entity-card
entity: water_heater.pentair_pool_pool
name: pool
tap_action:
  action: fire-dom-event
  browser_mod:
    service: browser_mod.popup
    data:
      content:
        type: custom:mushroom-climate-card
        entity: water_heater.pentair_pool_pool
        name: Pool Heater
        fill_container: false
        show_temperature_control: true
        collapsible_controls: false

**Comment by yestrdave** [comment]
![image](https://user-images.githubusercontent.com/93943061/198859493-97f6ed94-db35-4d1d-87e0-5193f10d4f24.png)

**Comment by mabsenger** [comment]
+1, would be great to have a water heater card!
